oat-sa/tao-core
# Fix/ADF-1664/Solar design glitches

### Related to: https://oat-sa.atlassian.net/browse/ADF-1664

### Summary

Fix a few UI glitches in the backoffice when the Solar Design applies and a custom theme is already in place.

### Details

Enforce some style to replace custom theme even when `!important` has been used.

Note: the issues arose especially with the extension taoStyles when a theme variant is applied.

### How to test

- check out the branch: `git checkout -t origin/fix/ADF-1664/solar-design-glitches`
- activate the feature flag: `php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_SOLAR_DESIGN_ENABLED -v true`
- log in to TAO backoffice
- look at the pages, compare with the UX mockups
